flink-ai-flow/ai_flow/plugin_interface/blob_manager_interface.py

```diff
@@ -18,9 +18,40 @@
 #
 from abc import ABC, abstractmethod
 from typing import Text, Dict
+
+from ai_flow.common.configuration import AIFlowConfiguration
 from ai_flow.common.module_load import import_string
 
 
+class BlobConfig(AIFlowConfiguration):
+
+    def __init__(self, config: Dict):
+        super().__init__()
+        self['blob_manager_class'] = None
+        self['blob_manager_config'] = {}
+        if config is None or 'blob_manager_class' not in config:
```

Comment:
Why `BlobConfig` inits with a Dict and checks correctness inner init(), while `SchedulerServiceConfig` inits with nothing and the caller needs to check correctness themselves ?

Reply:
I agree it is better to do check in the __init__ class but I think it is better to do such change in next pr which includes the refractor of scheduler plugin.